pkgs/by-name/lu/lurk/package.nix

```
@@ -10,8 +10,8 @@ rustPlatform.buildRustPackage rec {
 
   src = fetchFromGitHub {
     owner = "jakwai01";
-    repo = pname;
-    rev = "v${version}";
+    repo = "lurk";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
Nit. See https://github.com/NixOS/nixpkgs/pull/355973